docs/concepts/Package-Versioning.md

```diff
@@ -116,9 +116,10 @@ When referring to package dependencies, NuGet supports using interval notation f
 | [1.0,2.0) | 1.0 ≤ x < 2.0 | Mixed inclusive minimum and exclusive maximum version |
 | (1.0)    | invalid | invalid |
 
-### Examples
+### Best Practice
 
 Always specify a version or version range for package dependencies in project files, `packages.config` files, and `.nuspec` files. Without a version or version range, NuGet 2.8.x and earlier chooses the latest available package version when resolving a dependency, whereas NuGet 3.x and later chooses the lowest package version. Specifying a version or version range avoids this uncertainty.
```

Comment:
NuGet 2.8.x is out of support. Let's remove the whole of line 121.

Reply:
It's still important to specify a version.  I know PackageReference requires it.  I see nuspecs still allow dependencies without a version.  I think this can be reworded and avoid the mention of 2.8.x.

Reply:
How about: 

```suggestion
Always specify a version or version range for package dependencies in project files, `packages.config` files, and `.nuspec` files. Without a version or version range, when resolving a dependency, consistent restore results are not guaranteed.
```